### PR TITLE
Get compiling on Rust stable and update libs, Fix #1

### LINKED ISSUE #1: Stable compilation
Greets, this currently does not compile on stable due to the use of the `try_find` nightly-only feature.  It appears this is only used in one location, the `.try_find(|o| Ok(o.name()? == name))?` line in 341 in the current version of the source code in file wsdl.rs.  Any chance on replacing it with a stable equivalent, perhaps just a loop or so?  (I can submit a PR if wanted as I already have it patched locally.)

### PR DESCRIPTION
**Pull request by OvermindDL1**
This replaces try_find with just a for loop to do the same thing and updates the deps and gets the test/example in the README.md compiling again as it was broken before.

### CODE REVIEW

**Comment by OvermindDL1** [comment]
Also bumped the patch version to make ready to deploy to crates.io.

**Review by DrChat — APPROVED** [review]

**Comment by DrChat** [comment]
Thanks for the contribution!

Just noting that I haven't yet set up a CI pipeline to run in this repository, so I'm waiting until I have time to run a build test to merge this.
If you want to speed this along, you could copy a [CI pipeline](https://github.com/DrChat/SymProxyCloud-rs/blob/main/.github/workflows/ci.yaml) from one of my other repositories and include it in your pull request :)

**Comment by SamuelRamond** [comment]
Sorry to comment on a closed PR, do you think it would be possible to publish version `0.1.4` to `crates.io` ?
Thank you for the lib :)

**Pull request merged**
